enterprise/api_client/enterprise_catalog.py

```diff
@@ -125,7 +125,7 @@ def get_catalog_diff(self, enterprise_customer_catalog, content_keys, should_rai
         api_url = self.get_api_url(self.CATALOG_DIFF_ENDPOINT.format(catalog_uuid))
         body = {'content_keys': content_keys}
 
-        items_to_delete = {}
```

Comment:
Cleaning up a small code smell - I've tested the API and it is always an array, defaulting to a `dict` here was adding complexity elsewhere. Maybe this behaved differently in the past.